compiler/src/dmd/e2ir.d

```diff
@@ -7028,12 +7028,22 @@ elem *callCAssert(ref IRState irs, const ref Loc loc, Expression exp, Expression
             auto eassert = el_var(getRtlsym(RTLSYM.C__ASSERT_FAIL));
             ea = el_bin(OPcall, TYvoid, eassert, el_params(elmsg, efilename, eline, efunc, null));
         }
+        version (CRuntime_Glibc)
```

Comment:
```suggestion
        else version (CRuntime_Glibc)
```

Reply:
I tried this but serve-d was showing syntax errors, so I didn't try to compile it. This is what I meant to do.